clients/rust/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+mod generated;
+pub mod hash;
+pub mod util;
+
+pub use generated::programs::MPL_BUBBLEGUM_ID as ID;
+use generated::types::{LeafSchema, Version};
+pub use generated::*;
+use solana_program::keccak;
+
+// additional methods for the `LeafSchema` type
+
+impl LeafSchema {
```

Comment:
I think there is no much gain in having a `new` method that takes the exactly same arguments (fields) than the struct. It actually reduces the readability since it introduces an arbitrary order of the fields.

Reply:
I thought it was a bit more canonical Rust-like to provide a `new` associated function.  But that's fine with me.